ksql-engine/src/test/java/io/confluent/ksql/integration/WindowingIntTest.java

```diff
@@ -67,6 +66,13 @@
   private static final SessionWindowedDeserializer<String> SESSION_WINDOWED_DESERIALIZER =
       new SessionWindowedDeserializer<>(STRING_DESERIALIZER);
 
+  private static final Duration VERIFY_TIMEOUT = Duration.ofSeconds(60);
+
+  private static final long BATCH_0_SENT_MS = 1000000005001L;
```

Comment:
I've pulled out some constants for the time things are sent/produced and the windows they fit into.

I've also changed the base sent time from `1438997955143L`, to `1000000005001L`, as I find the latter, and its close cousins, a lot easier to grok when looking at logs.